ui/src/app/pages/access/data-access-requirements.spec.tsx

```diff
@@ -1935,7 +1935,7 @@ describe('DataAccessRequirements', () => {
       const expirationDate = new Date(oneYearFromNow());
       const formattedDate = expirationDate.toLocaleDateString('en-US', {
         year: 'numeric',
-        month: 'long',
+        month: 'short',
```

Comment:
This test was failing because we actually display the shortened month in the UI. It only appeared today because the change was merged during May which is still May when shortened.